packages/cryptoplease/lib/ui/tab_bar.dart

```diff
@@ -2,30 +2,73 @@ import 'package:flutter/material.dart';
 
 import 'colors.dart';
 
+enum CpTabBarVariant { dark, inverted }
+
 class CpTabBar extends StatelessWidget {
-  const CpTabBar({super.key, required this.tabs});
+  const CpTabBar({
+    super.key,
+    this.variant = CpTabBarVariant.dark,
+    required this.tabs,
+  });
 
+  final CpTabBarVariant variant;
   final List<Widget> tabs;
 
+  Color get _backgroundColor {
+    switch (variant) {
+      case CpTabBarVariant.dark:
+        return CpColors.darkBackground;
+      case CpTabBarVariant.inverted:
+        return CpColors.yellowColor;
+    }
+  }
+
+  Color get _indicatorBackground {
+    switch (variant) {
+      case CpTabBarVariant.dark:
+        return Colors.white;
+      case CpTabBarVariant.inverted:
+        return Colors.black;
+    }
+  }
+
+  Color get _labelColor {
+    switch (variant) {
+      case CpTabBarVariant.dark:
+        return Colors.black;
+      case CpTabBarVariant.inverted:
+        return Colors.white;
+    }
+  }
+
+  Color get _unselectedLabelColor {
+    switch (variant) {
+      case CpTabBarVariant.dark:
+        return CpColors.disabledTabColor;
+      case CpTabBarVariant.inverted:
+        return Colors.black;
+    }
+  }
```

Comment:
They can be made file-private.

Reply:
@rhbrunetto this is not resolved.

Reply:
They were using the `variant` property of the class to retrieve the color.
I've extracted them from the class in the latest commit and passed the variant as a parameter.

Please let me know if it's not properly fixed.